testdata/reference/assoc/assoc-with-dups.csv

```diff
@@ -95,9 +95,9 @@
 /c/ja/バイト/n	/c/ja/バイト	1.0	/d/jmdict	/r/SenseOf
 /c/ja/バイト/n	/c/en/computer_terminology	2.0	/d/jmdict	/r/HasContext
 /c/ja/バイト/n	/c/en/mathematics	2.0	/d/jmdict	/r/HasContext
-/c/ja/テスト	/c/ja/勉強	0.20000000000000018	/d/conceptnet/4/ja	/r/HasPrerequisite
+/c/ja/テスト	/c/ja/勉強	0.10000000000000009	/d/conceptnet/4/ja	/r/HasPrerequisite
```

Comment:
Why did the scores change?

Reply:
I changed the relative weights of nadya.jp and Yahoo Japan input, given that Yahoo Japan crowd-sourced a verification step, and nadya.jp was just random people playing a game.

Reply:
I don't see this change anywhere in the code (other than its effects) -- am I missing something?

Reply:
I suspect it was already merged but not accounted for in the tests, because the branches at this point were a complete mess.

Reply:
The change is in commit 71ba6b7, which I guess isn't in this PR.

Reply:
Thanks, got it.